### DISCUSSION TITLE
How to use GIT with Ghidra projects?

### DISCUSSION THREAD

**Discussion by alloc33** [NationalSecurityAgency/ghidra#5567] [Q&A]
I'm completely new in Ghidra and I'd like to know how to use version control with Ghidra projects if it's possible, use git diff-tool

**Comment by emteere** [comment]
Ghidra has it's own version control system called multi-user project repositories.  You can search the online help contents for "project repository" or from the help index GhidraProject->ProjectRepository.

If the project files were to be checked into git, they would be binary blobs.  Ghidra project repositories allow fine grained merging of changes/differences.
